paddle/phi/ops/yaml/legacy/static_ops.yaml

```
@@ -286,6 +286,18 @@
     func: fetch_barrier
   optional: x
 
+- op : flatten
+  args : (Tensor x, int start_axis, int stop_axis)
```

```suggestion
  args : (Tensor x, int start_axis = 1, int stop_axis = 1)
```
在原生ops.yaml里有默认值，这里是否应该对齐？下个PR会单独对齐